processing/src/main/java/org/apache/druid/query/filter/BoundDimFilter.java

```diff
@@ -610,13 +610,14 @@ private Supplier<DruidDoublePredicate> makeDoublePredicateSupplier()
     return Suppliers.memoize(doublePredicate);
   }
 
-  private static DruidLongPredicate makeLongPredicateFromBounds(
+  static DruidLongPredicate makeLongPredicateFromBounds(
```

Comment:
Is there a reason it cannot just be public?  You had reason to make it not private, shouldn't really hurt to take it fully public...

Reply:
oh, i re-used this method in `RangeFilter` which is in the same package to avoid duplicating the same logic, i guess can make it public though :shrug: